scripts/releaseChangelog.mjs

```diff
@@ -33,16 +34,30 @@ const MyOctokit = Octokit.plugin(retry);
 async function main(argv) {
   const { githubToken, ...other } = argv;
 
-  if (!githubToken) {
-    throw new TypeError(
-      'Unable to authenticate. Make sure you either call the script with `--githubToken $token` or set `process.env.GITHUB_TOKEN`. The token needs `public_repo` permissions.',
-    );
+  // Temporary code till everyone removes usage of GITHUB_TOKEN
+  /**
+   * @type {import('@octokit/rest').Octokit | undefined}
+   */
+  let octokit;
+  if (githubToken || process.env.GITHUB_TOKEN) {
+    const rl = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout,
+    });
+    const startMessage = githubToken
+      ? `You have provided a GitHub token via --githubToken. It is not recommended to pass it now since we will deprecate this flag in the future.`
+      : 'Found a value in "GITHUB_TOKEN" environment variable.\nIt is not recommended to use it and you should remove it\nfrom your shell rc file (.bashrc/.zshrc etc) if set there.';
+    const answer = (await rl.question(`${startMessage}\nDo you still want to use it? (y/N): `))
+      .trim()
+      .toLowerCase();
+    rl.close();
+    if (answer === 'y') {
```

Comment:
If the answer is 'N' shouldn't we exit the script? Currently it continues, right?